### PR TITLE
added ShoppingCart entity

### PR DESCRIPTION
**Pull request by TemkaHillfinger**
added CartItem entity

### CODE REVIEW

File: src/main/java/bookstore/service/cart/impl/ShoppingCartServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            newCartItem.setBook(book);
+            newCartItem.setQuantity(request.getQuantity());
+            cartItemRepository.save(newCartItem);
+            shoppingCart.getCartItems().add(newCartItem);
+        }
+        return shoppingCartMapper.toDto(shoppingCart);
+    }
+
+    @Override
+    public ShoppingCartDto updateCartItemQuantity(Long cartItemId, UpdateCartItemRequest request) {
```
Comment (MAMentorFX) [inline src/main/java/bookstore/service/cart/impl/ShoppingCartServiceImpl.java:65]: transaction

File: src/main/resources/db/changelog/changes/07-create-cart_items-table.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                  name: quantity
+                  type: INTEGER
+                  constraints:
+                    notNull: true
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```
Comment (MAMentorFX) [inline src/main/resources/db/changelog/changes/07-create-cart_items-table.yaml:35]:
```suggestion
                    nullable: false
                    
```

File: src/main/resources/db/changelog/changes/08-create-shopping-carts-table.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    notNull: true
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```
Comment (MAMentorFX) [inline src/main/resources/db/changelog/changes/08-create-shopping-carts-table.yaml:25]:
```suggestion
                    nullable: false
                    
```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -10,4 +10,8 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/05-create-categories-table.yaml
   - include:
-      file: db/changelog/changes/06-create-books_categories-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/06-create-books_categories-table.yaml
+  - include:
+      file: db/changelog/changes/07-create-cart_items-table.yaml
+  - include:
+      file: db/changelog/changes/08-create-shopping-carts-table.yaml
```
Comment (MAMentorFX) [inline src/main/resources/db/changelog/db.changelog-master.yaml:17]:
```suggestion
      file: db/changelog/changes/08-create-shopping-carts-table.yaml
      
```

**Review by MAMentorFX — CHANGES_REQUESTED** [review]

**Review by kozhukhovsky — APPROVED** [review]

**Pull request merged**